src/app/workflow-editor/object-inspector/io-inspector/workflow-io-inspector.component.ts

```diff
@@ -182,13 +306,13 @@ export class WorkflowIOInspectorComponent extends DirectiveBase implements OnIni
 
         this.tracked = this.form.controls["fileTypes"].valueChanges.subscribe((value) => {
             if (value) {
-                this.port.customProps["sbg:fileTypes"] = value;
+                this.port.fileTypes = value;
             } else {
-                delete this.port.customProps["sbg:fileTypes"];
+                delete this.port.fileTypes;
```

Comment:
Don't delete this property, set it to an empty array. The model will take care of serialization if the array is empty.